src/constants/WalletAndCurrencyConstants.ts

```diff
@@ -328,7 +328,8 @@ export const SPECIAL_CURRENCY_INFO: {
     chainCode: 'XMR',
     dummyPublicAddress: '46qxvuS78CNBoiiKmDjvjd5pMAZrTBbDNNHDoP52jKj9j5mk6m4R5nU6BDrWQURiWV9a2n5Sy8Qo4aJskKa92FX1GpZFiYA',
     isImportKeySupported: false,
-    unstoppableDomainsTicker: 'XMR'
+    unstoppableDomainsTicker: 'XMR',
+    maxSpendTargets: UTXO_MAX_SPEND_TARGETS
```

Comment:
I believe the maximum number of UTXOs for a Monero TX is 16.

Reply:
good catch. fixed